src/locales/en/translation.json

```diff
@@ -708,14 +708,18 @@
     "updateNameDateBirth": {
       "title": "Update your name or date of birth",
       "header": "Update your name or date of birth",
+      "titleCoi": "Update your full name or date of birth",
+      "headerCoi": "Update your full name or date of birth",
       "content": {
         "paragraph1": "To update your name or date of birth, you need to contact the GOV.UK One Login team.",
         "warningText": "You may not be able to use your GOV.UK One Login if your details are not up to date.",
         "warningTextFallback": "Warning",
         "radioButtonHeading": "What would you like to do?",
         "formRadioButtons": {
           "yes": "Contact the GOV.UK One Login team to update your name or date of birth",
-          "no": "Go back and check your details again"
+          "no": "Go back and check your details again",
+          "noHint": "",
```

Comment:
(I assume empty string here works as expected - and it doesn't generate a spurious hint element?)

Reply:
@Joe-Edwards-GDS Correct, the hint element is not added if it is blank